libparsec/crates/platform_storage/src/web/mod.rs

```diff
@@ -1,5 +1,11 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
 
+use libparsec_types::VersionInt;
+
 pub mod certificates;
+mod db;
+mod model;
 pub mod user;
 pub mod workspace;
+
+const DB_VERSION: VersionInt = 1;
```

Comment:
I don't need to need to use `VersionInt` for `DB_VERSION`, that value is internal to the web implementation.
I would suggest to keep it simple and use `u32` directly :wink:

Reply:
:+1: